services/hexathons/src/routes/visit.ts

```diff
@@ -0,0 +1,138 @@
+/* eslint-disable no-underscore-dangle */
+import { apiCall, asyncHandler, BadRequestError, checkAbility } from "@api/common";
+import { Service } from "@api/config";
+import express from "express";
+import { FilterQuery } from "mongoose";
+
+import { VisitModel, Visit } from "../models/visit";
+
+export const visitRouter = express.Router();
```

Comment:
could we rename this to `sponsorVisitRouter` to keep consistency with the route name? I like the name change u made